.github/CODEOWNERS

```
@@ -1 +1,3 @@
-* @elastic/elastic-agent-control-plane
\ No newline at end of file
+* @elastic/elastic-agent-control-plane
+/.github/actions @elastic/observablt-ci @elastic/observablt-ci-contractors
+/.github/workflows @elastic/observablt-ci @elastic/observablt-ci-contractors
```

I'm inclined to remove the contractors from the codeowners for now

```suggestion
/.github/actions @elastic/observablt-ci
/.github/workflows @elastic/observablt-ci
```